src/__tests__/gilded_rose_spec.js

```diff
@@ -84,6 +114,33 @@ describe("Gilded Rose", function() {
           expect(shop.items[0].quality).toEqual(22);
         });
 
+        it("does not go over 50 when quality is 49 and sellIn is less than 11", () => {
+          const items = [
+            new Item("Backstage passes to a TAFKAL80ETC concert", 10, 49)
+          ];
+          const shop = new Shop(items);
+          shop.updateQuality();
+          expect(shop.items[0].quality).toEqual(50);
+        });
+
+        it("does not go over 50 when quality is 49 and sellIn is less than 6", () => {
+          const items = [
+            new Item("Backstage passes to a TAFKAL80ETC concert", 5, 49)
+          ];
+          const shop = new Shop(items);
+          shop.updateQuality();
+          expect(shop.items[0].quality).toEqual(50);
+        });
+
+        it("quality increases by two sellIn is less than 10", () => {
```

Comment:
If 11 is the sellIn and it gets reduced by 1 then it is 10 and not less than 10. So either test name is wrong or given sellIn